pkgs/applications/video/filebot/default.nix

```diff
@@ -0,0 +1,58 @@
+{ stdenv, fetchurl, openjdk8, makeWrapper, autoPatchelfHook
+
+, zlib, libzen, libmediainfo, curl, libmms, glib
+}:
+
+let
+  curlWithGnuTls = curl.override { gnutlsSupport = true; sslSupport = false; };
+
+in
+
+stdenv.mkDerivation rec {
+  name = "${pname}-${version}";
+  pname = "filebot";
+  version = "4.8.5";
+
+  src = fetchurl {
+    # This is a closed-source application, hence downloading a binary package.
+    url = "https://get.filebot.net/filebot/FileBot_4.8.5/FileBot_4.8.5-portable.tar.xz";
+    sha256 = "1c2a97rkv57cvy4arnmw9f3zdvxwb5zfvr9iqgpn9rkda00f2hy8";
+  };
+
+  unpackPhase = ''tar xvf $src'';
+
+  nativeBuildInputs = [ makeWrapper autoPatchelfHook
+                        # The following are lib dependencies for autoPatchelfHook to work.
+                        stdenv.cc.cc.lib zlib libzen libmediainfo curlWithGnuTls libmms glib ];
```

Comment:
These should be buildInputs (runtime dependencies), not nativeBuildInputs (build dependencies).